plugins/modules/aci_bfd_multihop_interface_profile.py

```diff
@@ -0,0 +1,361 @@
+        l3out_logical_interface_profile=dict(type="str", aliases=["logical_interface_profile_name", "logical_interface_profile"]),
+        name=dict(type="str", aliases=["bfd_multihop_interface_profile"]),
+        name_alias=dict(type="str"),
+        description=dict(type="str", aliases=["descr"]),
+        interface_profile_type=dict(type="str", choices=["none", "sha1"]),
+        key=dict(type="str", no_log=True),
+        key_id=dict(type="int"),
+        bfd_multihop_interface_policy=dict(type="str", aliases=["multihop_interface_policy", "multihop_interface_policy_name"]),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["tenant", "l3out", "l3out_logical_node_profile", "l3out_logical_interface_profile", "bfd_multihop_interface_policy"]],
+            ["state", "present", ["tenant", "l3out", "l3out_logical_node_profile", "l3out_logical_interface_profile"]],
+        ],
+    )
+
+    tenant = module.params.get("tenant")
+    l3out = module.params.get("l3out")
+    l3out_logical_node_profile = module.params.get("l3out_logical_node_profile")
+    l3out_logical_interface_profile = module.params.get("l3out_logical_interface_profile")
+    name = module.params.get("name")
+    name_alias = module.params.get("name_alias")
+    description = module.params.get("description")
+    interface_profile_type = module.params.get("interface_profile_type")
+    key = module.params.get("key")
+    key_id = module.params.get("key_id")
+    bfd_multihop_interface_policy = module.params.get("bfd_multihop_interface_policy")
+    state = module.params.get("state")
+
+    aci = ACIModule(module)
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fvTenant",
+            aci_rn="tn-{0}".format(tenant),
+            module_object=tenant,
+            target_filter={"name": tenant},
+        ),
+        subclass_1=dict(
+            aci_class="l3extOut",
+            aci_rn="out-{0}".format(l3out),
+            module_object=l3out,
+            target_filter={"name": l3out},
+        ),
+        subclass_2=dict(
+            aci_class="l3extLNodeP",
+            aci_rn="lnodep-{0}".format(l3out_logical_node_profile),
+            module_object=l3out_logical_node_profile,
+            target_filter={"name": l3out_logical_node_profile},
+        ),
+        subclass_3=dict(
+            aci_class="l3extLIfP",
+            aci_rn="lifp-{0}".format(l3out_logical_interface_profile),
+            module_object=l3out_logical_interface_profile,
+            target_filter={"name": l3out_logical_interface_profile},
+        ),
+        subclass_4=dict(
+            aci_class="bfdMhIfP",
+            aci_rn="bfdMhIfP",
+            module_object="bfdMhIfP",
+            target_filter={"name": ""},
```

Comment:
Wouldn't be better to do `target_filter={"name": l3out_logical_interface_profile },`?